packages/firestore/test/unit/specs/listen_spec.test.ts

```diff
@@ -141,6 +141,19 @@ describeSpec('Listens:', [], () => {
     }
   );
 
+  specTest("Doesn't raise events for non-active target", [], () => {
```

Comment:
The change that's coming is the sequence where userListens, userUnlistens, watchRemoves(with error),  watchRemoves(no error).

As specified this test is non-representative of the real system because Watch should respond to every request we make (or fail the stream).